### PR TITLE
Bugfix 5964/Fix UGNT footnotes

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
- updated to latest usfm-js with UGNT footnotes fix, which also required updating word-aligner, tc-ui-toolkit, and tc-source-content-updater.
- regenerated resource content for uhb and ugnt with this footnote fix.

#### Please include detailed Test instructions for your pull request:
- open up an Acts project in WA.  In scripture pane, Should see warning in Acts 8:37 in UGNT that there is no text for this verse.

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/5967)

### CODE REVIEW

**Comment by mannycolon** [comment]
Hey you deleted the translationWords resources that have the group data for tW tool

File: tcResources/grc/bibles/ugnt/v0.5/index.json [hunk 1]
```diff
@@ -561,7 +561,7 @@
     "16": {
       "1": 15,
       "2": 5,
-      "3": 23,
+      "3": 0,
```
Comment (mannycolon) [inline tcResources/grc/bibles/ugnt/v0.5/index.json:564]: So we have an empty verse for 16:3?

Comment (PhotoNomad0) [reply tcResources/grc/bibles/ugnt/v0.5/index.json:564]: This is a footnoted verse (all content is in footnote) - new word count should be 0.

Comment (PhotoNomad0) [reply tcResources/grc/bibles/ugnt/v0.5/index.json:564]: I mean alignment count

**Review by mannycolon — APPROVED** [review]

**Pull request merged**